daft-derive/src/internals/imp.rs

```diff
@@ -187,10 +183,12 @@ impl Visit<'_> for BanDaftAttrsVisitor<'_> {
     }
 }
 
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
```

Comment:
This is a nice organization.

Reply:
Yeah -- this is kind of how syn visitors are meant to be used, fwiw.